test/controllers/EditSetControllerSpec.scala

```diff
@@ -1148,48 +1161,6 @@ class EditSetControllerSpec extends BaseSpec {
             )
 
           }
-          "doesn't exist" in {
```

Comment:
Why was this scenario removed?

Reply:
The intent of that scenario is to show what happens when the start date doesn't exist - not the record (by OCI).

If it did refer to a record that didn't exist, that was accidental.

See the one below that for end date (line 1239).

Reply:
Actually, you just need to change this line in the original scenario:

```
val result = submitWhileLoggedIn(1, "COAL.2022.V1RJW.P", values)
```

Incidentally, it was edit set ID `2` which works, but is misleading so best to change that too.

Reply:
Changed; I misunderstood the original test and though it was asserting there was a bad response for a missing OCI.

Reply:
Confirmed.